### PR TITLE
feat: allow running sidero without siderolink, changes after org rename

### PR DESCRIPTION
**Pull request by Unix4ever**
Pause siderolink container if it fails to set up the network.
Also update `talos-systems` references in Makefile`, `Dockerfile`, all
docs and tests.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 REGISTRY ?= ghcr.io
-USERNAME ?= talos-systems
+USERNAME ?= siderolink
```
Comment (frezbo) [inline Makefile:2]: shouldn't this be `siderolabs` :sweat_smile:

File: Makefile [hunk 2]
```diff
@@ -148,7 +148,7 @@ release-notes:
 
 $(ARTIFACTS)/$(TALOS_RELEASE)/%:
 	@mkdir -p $(ARTIFACTS)/$(TALOS_RELEASE)/
-	@curl -L -o "$(ARTIFACTS)/$(TALOS_RELEASE)/$*" "https://github.com/talos-systems/talos/releases/download/$(TALOS_RELEASE)/$*"
+	@curl -L -o "$(ARTIFACTS)/$(TALOS_RELEASE)/$*" "https://github.com/siderolink/talos/releases/download/$(TALOS_RELEASE)/$*"
```
Comment (frezbo) [inline Makefile:151]: same here

File: website/content/docs/v0.1/Guides/bootstrapping.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Talos Systems and the Cluster API community have created tools to help make this
 First, you need to install the latest `talosctl` by running the following script:
 
 ```bash
-curl -Lo /usr/local/bin/talosctl https://github.com/talos-systems/talos/releases/latest/download/talosctl-$(uname -s | tr "[:upper:]" "[:lower:]")-amd64
+curl -Lo /usr/local/bin/talosctl https://github.com/siderolink/talos/releases/latest/download/talosctl-$(uname -s | tr "[:upper:]" "[:lower:]")-amd64
```
Comment (frezbo) [inline website/content/docs/v0.1/Guides/bootstrapping.md:21]: `siderolink` -> `siderolabs` :sweat_smile:

File: website/content/docs/v0.1/Guides/bootstrapping.md [hunk 4]
```diff
@@ -196,7 +196,7 @@ metadata:
   name: default
 spec:
   kernel:
-    url: "https://github.com/talos-systems/talos/releases/latest/download/vmlinuz-amd64"
+    url: "https://github.com/siderolink/talos/releases/latest/download/vmlinuz-amd64"
```
Comment (frezbo) [inline website/content/docs/v0.1/Guides/bootstrapping.md:199]: same here

File: website/content/docs/v0.1/Guides/bootstrapping.md [hunk 5]
```diff
@@ -216,7 +216,7 @@ spec:
       - talos.platform=metal
       - talos.config=http://$PUBLIC_IP:9091/configdata?uuid=
   initrd:
-    url: "https://github.com/talos-systems/talos/releases/latest/download/initramfs-amd64.xz"
+    url: "https://github.com/siderolink/talos/releases/latest/download/initramfs-amd64.xz"
```
Comment (frezbo) [inline website/content/docs/v0.1/Guides/bootstrapping.md:219]: same

File: website/content/docs/v0.1/Overview/introduction.md [hunk 6]
```diff
@@ -4,10 +4,10 @@ weight: 1
 title: Introduction
 ---
 
-Sidero ("Iron" in Greek) is a project created by the [Talos Systems](https://www.talos-systems.com/) team.
+Sidero ("Iron" in Greek) is a project created by the [Talos Systems](https://www.siderolink.com/) team.
```
Comment (frezbo) [inline website/content/docs/v0.1/Overview/introduction.md:7]:
```suggestion
Sidero ("Iron" in Greek) is a project created by the [Sidero Labs](https://www.sideroabs.com/) team.

```

**Review by frezbo — CHANGES_REQUESTED** [review]
`siderolink` -> `siderolabs` in many places

**Comment by Unix4ever** [comment]
ok, I messed up. Let me fix that :))

**Comment by smira** [comment]
What is the core issue with SideroLink in docker, why does it fail to start up?

**Comment by Unix4ever** [comment]
> What is the core issue with SideroLink in docker, why does it fail to start up?
Oh I was actually able to make it work now. So it looks like you just need to enable wireguard kernel module and reboot

File: website/content/docs/v0.1/Overview/introduction.md [hunk 7]
```diff
@@ -4,10 +4,10 @@ weight: 1
 title: Introduction
 ---
 
-Sidero ("Iron" in Greek) is a project created by the [Talos Systems](https://www.talos-systems.com/) team.
+Sidero ("Iron" in Greek) is a project created by the [Siderolabs](https://www.siderolabs.com/) team.
 The goal of this project is to provide lightweight, composable tools that can be used to create bare-metal Talos + Kubernetes clusters.
 These tools are built around the Cluster API project.
-Sidero is also a subproject of Talos Systems' [Arges](https://github.com/talos-systems/arges) project, which will publish known-good versions of these components (along with others) with each release.
+Sidero is also a subproject of Siderolabs' [Arges](https://github.com/siderolabs/arges) project, which will publish known-good versions of these components (along with others) with each release.
```
Comment (smira) [inline website/content/docs/v0.1/Overview/introduction.md:10]: should we remove Arges for now?

**Pull request closed without merging**